File: Swift/Two-Sum.swift
class Solution {
    func twoSum(_ nums: [Int], _ target: Int) -> [Int] {
        var dictOfIndices = [Int:Int]()
        var resultArray : [Int] = [Int]()
        for (index, value) in nums.enumerated() {
            let indicesValue = target - nums[index]

            if let foundIndex = dictOfIndices[indicesValue] {
                resultArray.append(foundIndex )
                resultArray.append(index)
                return resultArray
            }
            dictOfIndices[value] = index
        }
        return resultArray
    }
}

File: Python/Two-Sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        dct = {}  # creating an empty dictionary 
        for index,value in enumerate(nums):
            indices = target - nums[index]
            if indices in dct:
                return (dct[indices], index)
            dct[value] = index    
